spec/request/in_progress_forms_request_spec.rb

```diff
@@ -84,19 +84,20 @@
         end
 
         context 'with the x key inflection header set' do
-          let(:form_data) do
-            { 'view:hasVaMedicalRecords' => true }
-          end
-
           it 'converts the json keys' do
+            form_data = { 'view:hasVaMedicalRecords' => true }
+
             in_progress_form.update(form_data: form_data)
             get v0_in_progress_form_url(in_progress_form.form_id),
                 params: nil,
                 headers: { 'HTTP_X_KEY_INFLECTION' => 'camel' }
-            expect(response.body).to eq({
-              form_data: form_data,
-              metadata: in_progress_form.metadata
-            }.deep_transform_keys { |key| key.to_s.underscore.camelize(:lower) }.to_json)
+            body = JSON.parse(response.body)
+            expect(body.keys).to include('formData', 'metadata')
+            # using VA as Va is deprecated, when forms are updated to use VA, this test should reflect that
```

Comment:
How/why is the deprecation happening?

Reply:
it's not.  this comment is old. good catch!